keps/sig-node/2727-grpc-probe/README.md

```diff
@@ -354,10 +436,24 @@ Feature is promoted to beta in 1.24.
 
 Feature is promoted to GA in 1.27.
 
+## Drawbacks
+
+<!--
```

Comment:
* gRPC is a specific example of a higher-layer protocol. If we implement this, why not other RPC systems?

Rebuttal: Demand for gRPC is high in the cloud-native space and it is already used by Kubernetes, so the incremental cost is low.  If there were demand for other RPC systems, we would consider it.

Reply:
updated, thanks